### PR TITLE
- Separate the AmpHandler and RedirectHandler logic into individual f…

### PR DESCRIPTION
**Pull request by apodi**
**Refactor AmpHandler and RedirectHandler into separate modules**

 - This separation allows for easier maintenance and testing of the individual handlers.
 - Each handler is responsible for a specific task, making the codebase more organized and maintainable.
- Update the main clean function to utilize the new handler modules, enhancing clarity and testability of the codebase.

### CODE REVIEW

**Comment by DrKain** [comment]
Please remove the tests, they are not needed at this time.

File: data/supported-sites.txt
Comment (DrKain) [inline data/supported-sites.txt:1]: When adding new websites it's best to add one website per commit, or if you're adding multiple websites list them in the commit description. Please remove these for now.

**Comment by DrKain** [comment]
Did you use Copilot for this?

**Comment by apodi** [comment]
No i did not use Copilot

File: data/supported-sites.txt
Comment (apodi) [reply data/supported-sites.txt:1]: reverted all

**Comment by apodi** [comment]
The pull request has been updated

**Comment by DrKain** [comment]
Thanks. I'll give it another review when I get the time.

**Pull request closed without merging**